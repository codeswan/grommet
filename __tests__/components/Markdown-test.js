// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react/lib/ReactTestRenderer';

import Markdown from '../../src/js/components/Markdown';

describe('Markdown', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Markdown content='test'
        components={{
          p: { props: { className: 'testing', size: 'large' } }
        }} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
