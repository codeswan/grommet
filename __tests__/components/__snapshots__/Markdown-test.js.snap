exports[`Markdown has correct default options 1`] = `
<p
  className="grommetux-paragraph testing grommetux-paragraph--large"
  dangerouslySetInnerHTML={undefined}
  id={undefined}>
  test
</p>
`;
